### PR TITLE
Switch WinUI to the Windows 11 styles

### PR DESCRIPTION
**Pull request by PankajBhojwani**

### CODE REVIEW

**Comment by zadjii-msft** [comment]
* [ ] How are we lanning on dealing with the numerous TabView quirks in v2? Just ship it as-is and fix in post? Or manually adjust the styling now?
  * [ ] EX the close button hover is weirdly wide
  * [ ] the tab text is bold
* [ ] Icons throughout the app are still on the Segoe MDL2, not Segoe Fluent
* [x] Tab color picker looks fine
* [x] Tab renamer looks fine (if maybe needs a little padding...)
* [x] Window Renamer / ID toast looks fine
* [ ] About looks... Fine? The links aren't underlined anymore, but that might just be how WinUI works these days
* [ ] `InfoBar`s look totally ~~insane~~ transparent 
image

File: src/cascadia/TerminalApp/TerminalPage.xaml [hunk 1]
```diff
@@ -12,6 +12,15 @@
       Background="Transparent"
       mc:Ignorable="d">
 
+    <Page.Resources>
+        <!--  Info Bar  -->
+        <Style x:Key="SettingsInfoBarStyle"
+               TargetType="mux:InfoBar">
+            <Setter Property="Margin" Value="-2" />
```
Comment (PankajBhojwani) [inline src/cascadia/TerminalApp/TerminalPage.xaml:19]: For some reason there's a bit of space all around the info bar that the background will not cover (and is transparent), hence the negative margin

Comment (zadjii-msft) [reply src/cascadia/TerminalApp/TerminalPage.xaml:19]: We should probably file an issue upstream that Informational `InfoBar`s have a transparent BG

Comment (zadjii-msft) [reply src/cascadia/TerminalApp/TerminalPage.xaml:19]: In [Controls v1, an `InfoBar`](https://github.com/microsoft/microsoft-ui-xaml/blob/9052972906c8a0a1b6cb5d5c61b27d6d27cd7f11/dev/InfoBar/InfoBar_themeresources_v1.xaml#L13) has either `#2f2f2f` or `#2B2B2B` as the background, which are both fully opaque

In In [Controls v2, an `InfoBar`](https://github.com/microsoft/microsoft-ui-xaml/blob/775568ceb6ef0821b27b49b0b16f92270e8df65f/dev/InfoBar/InfoBar_themeresources.xaml#L14) uses [`SystemFillColorAttentionBackgroundBrush`](https://github.com/microsoft/microsoft-ui-xaml/blob/5673d0ca4b6b0e4645637bbab37f48f4825baec3/dev/CommonStyles/Common_themeresources_any.xaml#L100) which is `#08FFFFFF`, so it's partially transparent (mostly transparent actually).

Comment (zadjii-msft) [reply src/cascadia/TerminalApp/TerminalPage.xaml:19]: Ah it's already in discussion... https://github.com/microsoft/microsoft-ui-xaml/issues/5741

Comment (DHowett) [reply src/cascadia/TerminalApp/TerminalPage.xaml:19]: I would rather give the thing that hosts the infobar a background than weirdly expand its margins.......

Comment (DHowett) [reply src/cascadia/TerminalApp/TerminalPage.xaml:19]: (AKA: this feels like a hack)

**Comment by DHowett** [comment]
How does it look overall? Should we merge and test in post?

**Comment by PankajBhojwani** [comment]
> Should we merge and test in post?

I like this idea, the visual issues we've found so far have been fixed getting it in will help us find more (if any) faster

**Comment by DHowett** [comment]
image
renamed the PR!

**Comment by DHowett** [comment]
> * [ ]  How are we lanning on dealing with the numerous TabView quirks in v2? Just ship it as-is and fix in post? Or manually adjust the styling now?
>   
>   * [ ]  EX the close button hover is weirdly wide
>   * [ ]  the tab text is bold

Ship, file, fix. :)

**Review by DHowett — APPROVED** [review]

**Review by zadjii-msft — APPROVED** [review]

**Pull request merged**